src/site/_collections/authors.js

```diff
@@ -17,6 +17,7 @@
 const authorsData = require('../_data/authorsData.json');
 const {livePosts} = require('../_filters/live-posts');
 const setdefault = require('../_utils/setdefault');
+const {sortByDate} = require('../_utils/sort-by-date');
```

Comment:
Should this function be renamed to something like `sortByUpdated` as that's the primary thing it's trying to sort on. It only uses `date` as a fallback.

Reply:
I'll rename it then merge